.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
FYI that I'm not going to want to commit this file in the PR (though I obviously understand why you have it running locally)

Reply:
i deleted it, didn't mean to let it be there at the very end just used for transition between computers